src/general/GameWorld.cs

```diff
@@ -82,6 +82,9 @@ public GameWorld()
     [JsonProperty]
     public Species PlayerSpecies { get; private set; } = null!;
 
+    [JsonIgnore]
+    public Dictionary<uint, Species> Species => worldSpecies;
```

Comment:
Would it be possible to return a read only dictionary interface? It would be extremely bad if some external code modified the existing species data through this.